flink-staging/flink-table/pom.xml

```diff
@@ -37,7 +37,7 @@ under the License.
 		<dependency>
 			<groupId>com.google.guava</groupId>
 			<artifactId>guava</artifactId>
-			<version>${guava.version}</version>
+			<version>14.0.1</version>
```

Comment:
Why do you manually set the version here? Is it necessary because of something with HCat? @rmetzger what could we do in such a case?